github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1540,3 +1540,34 @@ presubmits:
           resources:
             requests:
               memory: "4Gi"
+  - name: pull-kubevirt-check-unassigned-tests
+    cluster: ibm-prow-jobs
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: true
+    skip_report: false
```

Comment:
Not sure whether we should at first start without reporting and make it optional until all tests are assigned to a SIG? Otherwise we'll directly block all merges right now.

Reply:
Makes total sense, updated PTAL.

Reply:
Now that all the tests in master are assigned to a SIG I've used changes to always run this lane, set it to report and required. IT will prevent new tests to be added without any sig, like in this PR https://github.com/kubevirt/kubevirt/pull/5402#pullrequestreview-678246698